lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -176,45 +176,55 @@ sub biotype_groups : Path("biotypes/groups") Args(0) ActionClass('REST') {
 
 sub biotype_group_GET { }
 
-sub biotype_group : Path("biotypes/groups") Args(1) ActionClass('REST') {
-  my ($self, $c, $group) = @_;
+sub biotype_group : Path("biotypes/groups") CaptureArgs(2) ActionClass('REST') {
+  my ($self, $c, $group, $object_type) = @_;
+
+  $c->go('ReturnError', 'custom', ["Missing mandatory argument ':group' for endpoint info/biotypes/groups/:group/:object_type"]) unless $group;
 
   my $biotypes;
 
   try {
-    $biotypes = $c->model('Biotype')->fetch_biotypes_by_group($group);
+    $biotypes = $c->model('Biotype')->fetch_biotypes_by_group($group, $object_type);
 
   } catch {
     $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
     $c->go('ReturnError', 'custom', [qq{$_}]);
   };
 
   ## Return 404 for get requests with no return
-  $c->go( 'ReturnError', 'not_found', ["biotypes not found for group $group"]) unless $biotypes->[0];
+  if ( !$biotypes->[0] ) {
```

Comment:
A similar observation here. I always get nervous around direct element access without checking it exists first.